ReplayKitExample/README.md

```diff
@@ -14,11 +14,18 @@ Use an `RPBroadcastSampleHandler` to receive audio and video samples. Video samp
 
 An iOS 12.0 extension is not limited to capturing the screen of a single application. In fact, it is possible to capture video from any application including the home screen.
 
-In order to reduce memory usage, the extension configures `ReplayKitVideoSource` to downscsale incoming video frames, and prefers the H.264 video codec. In a Group Room, the extension connects as a publish-only Participant ([TVIConnectOptionsBuilder.automaticSubscriptionEnabled](https://twilio.github.io/twilio-video-ios/docs/latest/Classes/TVIConnectOptionsBuilder.html#//api/name/automaticSubscriptionEnabled)) to further reduce bandwidth, memory, and CPU requirements.
+In order to reduce memory usage, the extension configures `ReplayKitVideoSource` to downscsale incoming video frames, and prefers the H.264 video codec by default. In a Group Room, the extension connects as a publish-only Participant ([TVIConnectOptionsBuilder.automaticSubscriptionEnabled](https://twilio.github.io/twilio-video-ios/docs/latest/Classes/TVIConnectOptionsBuilder.html#//api/name/automaticSubscriptionEnabled)) to further reduce bandwidth, memory, and CPU requirements.
 
 **ReplayKitVideoSource**
 
-This `TVIVideoSource` produces `TVIVideoFrame`s from `CMSampleBuffer`s captured by ReplayKit. In order to reduce memory usage, this class may be configured (via a format request) to downscale the captured content.
+This `TVIVideoSource` produces `TVIVideoFrame`s from `CMSampleBuffer`s captured by ReplayKit. In order to reduce memory usage, instances may be configured (via a format request) to downscale the captured content.
+
+The source offers several capabilities to optimize for different use cases:
+
+1. Screencast mode (In-App Conferencing). This setting preserves spatial resolution (detail) over all else. The input from ReplayKit is always capped at 15 fps, and is not downscaled.
+2. Video mode (Broadcast Extension). This setting balances spatial and temporal resolution, and is content aware. When app content is shown, the input is capped at 15 fps. However, when playback of 24 fps video content is detected the input cap is raised to preserve the native cadence of the video.
+3. Inverse Telecine (IVTC) for 24 frames / second with 3:2 pulldown. Some apps (that do not use AVPlayer for playback) perform a telecine by drawing the same video frame to screen multiple times. These duplicate frames waste bandwidth, and CPU while making it more likely for subscribers to miss vsyncs in their own display pipeline. When 30 frame / second input with 3:2 pulldown is detected, the IVTC algorithm removes the duplicate frames to restore the content's natural frame rate.
```

Comment:
I literally had to google IVTC, can we add link to some documentation to learn about IVTC.